server/public/styles.css

```diff
@@ -0,0 +1,18 @@
+.profile-picture {
+  width: 225px;
+  height: 225px;
+}
+
+@media only screen and (min-width: 725px) {
+  .text-align-center {
+    text-align: start;
+  }
+
+  .mobile-row {
+    display: flex;
+  }
+
+  .profile-picture {
+    margin-right: 1.5rem;
+  }
```

Comment:
Instead of applying a margin-right just to the picture, put the check profile content in your `.container`